C/CUTEst/build_tarballs.jl

```diff
@@ -3,7 +3,7 @@
 using BinaryBuilder
 
 name = "CUTEst"
-version = v"2.0.4"
+version = v"2.0.5"
```

Comment:
You didn't change the sources?

Reply:
Just the build_tarballs, which will affect the jll. Should I not modify the version number?

Reply:
https://juliapackaging.github.io/BinaryBuilder.jl/dev/building/#Version-number

Reply:
Thanks.

Reply:
We're probably going to change the versioning scheme soon, but for the time being better to keep consistent